src/main/java/com/google/gerrit/extensions/api/changes/Changes.java

```diff
@@ -74,7 +75,7 @@ ChangeApi id(String project, String branch, String id)
     public abstract List<ChangeInfo> get() throws RestApiException;
 
     public QueryRequest withQuery(String query) {
-      this.query = query;
+      this.query = StandardKeyEncoder.encode(query);
```

Comment:
Unfortunately not. E.g. if an old-style query like "is:open+is:watched" is used, with literal "+", that would be encoded as "is%3Aopen%2Bis%3Awatched", i.e. the "+" would be encoded as "%2B".

I see how break breaking backwards compatibility is bad. What do you think about it, should we abandon this change altogether, or introduce a new "query2"-style method?